src/pages/Home/index.tsx

```diff
@@ -9,13 +9,19 @@ import BobaModalForm from './CustomizeBobaModalBody';
 import Button from 'react-bootstrap/Button';
 import useApi from 'hooks/API';
 import Card from 'react-bootstrap/Card';
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
   const BACKEND_API = process.env.REACT_APP_BOBA_FETCH;
-  const { data, error, loading } = useApi(BACKEND_API as string, '/boba_list');
-  const { data: toppingsList } = useApi(BACKEND_API as string, '/toppings_list');
 
-  const { data: milkList } = useApi(BACKEND_API as string, '/milk_list');
+  const client = new ApolloClient({
+    uri: BACKEND_API + '/boba_list',
+    cache: new InMemoryCache(),
+  });
+
+  const { data, error, loading } = useApi(BACKEND_API as string, '/boba_list');
+  const { data: toppingsList, error: toppingsError } = useApi(BACKEND_API as string, '/boba_list');
+  const { data: milkList, error: milkError } = useApi(BACKEND_API as string, '/boba_list');
```

Comment:
why milk and toppings are all hitting `/boba_list` endpoint?

Reply:
i have to put something because when i fix the GraphQL i remove these route at the backend so they wont be returning anything

Reply:
Is your customize modal still working since all milk and topping are targeting boba_list which is entirely different data sets

Reply:
> i have to put something because when i fix the GraphQL i remove these route at the backend so they wont be returning anything
You're ignoring the regressions failures by disabling topping and milk endpoints, which doesn't resolve the issue. Original core functionalities should not break.
If needed, you can create a second custom hook like useAPI_v2 for graphQL only, while maintaining the old useAPI for migrating topping/milk endpoints later

Reply:
![image](https://user-images.githubusercontent.com/103973779/236600808-e78351a8-7405-4985-8e41-09d7412a4d07.png)
i mean when i disable just toppinglist and milklist , it start yelling because we are using that in one of our handle

Reply:
What is it yelling? be more descriptive. Your screenshot doesn't mention any error only red lines.
We shouldn't have to disable to touch those topping/milk endpoints since you're only suppose to be working with bobalist only. If boba_list is somehow dependent on toppings/milk, then the original implementation is flawed and should be separated so it doesn't break.